mutation_reader.cc

```diff
@@ -811,9 +811,9 @@ mutation_source make_combined_mutation_source(std::vector<mutation_source> adden
         std::vector<flat_mutation_reader> rd;
```

Comment:
I think this function should be extended to also take `mutation_reader::forwarding` and then pass that down to the `make_reader` calls below.